tests/test_collections.py

```diff
@@ -129,6 +129,7 @@ def fetch_collection_paging_response(self, codes, replica: str, per_page: int):
 
     def test_collection_paging(self):
         min_page = 10
+        time.sleep(4)  # wait for collection replicas to sync
         self.create_temp_user_collections(num=min_page + 1)  # guarantee at least one paging response
```

Comment:
This solution may make this test less intermittent, but highlights a more fundamental issue.

Can test dependency on sync daemon success be removed by making `create_temp_user_collections` replica specific?

Reply:
That sounds good to me.  Should make it shorter too and I'm always for shorter tests.

Reply:
Done.